templates/web/fixmystreet.com/about/information-for-councils.html

```diff
@@ -58,11 +58,11 @@ <h1>Information for councils</h1>
 <dd>
 <p>FixMyStreet can be connected with your existing system(s) in a number of ways:
 <ol>
-<li>You can do this yourself (for free) using our Open311 API</li>
-<li>We can build and maintain the integration for you (carries an annual fee)</li>
+<li>You can do this yourself (for free) using our <a href="https://www.fixmystreet.com/about/open311-api-info">Open311 API</a>.</li>
```

Comment:
Don't need to specify the domain here (so that e.g. it works on staging), can just be "/about/open311-api-info"

Reply:
Done (hopefully)